sysutils/cdrtools/Portfile

```diff
@@ -1,98 +1,205 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem          1.0
-PortGroup           compiler_blacklist_versions 1.0
-
-name                cdrtools
-version             3.01
-revision            1
-categories          sysutils
-# listing all these licenses is somewhat redundant when taken as a whole,
-# but different separable components are licensed differently
-license             CDDL-1 BSD LGPL-2.1 GPL-2+ GPL-2
-platforms           darwin
-conflicts           dvdrtools
-
-description         ISO 9660 filesystem and CD creation tools
-long_description    The cdrtools software includes programs to create \
-                    and/or extract ISO 9660 filesystems, verify their \
-                    integrity, and write them to a disc.
-homepage            http://cdrecord.org
-maintainers         nomaintainer
-master_sites        sourceforge:project/cdrtools
-
-checksums           rmd160  3696b1bc502905cbe7c06e492b42d2ffe2dba982 \
-                    sha256  ed282eb6276c4154ce6a0b5dee0bdb81940d0cbbfc7d03f769c4735ef5f5860f \
-                    size    2087416
-
-use_bzip2           yes
-
-depends_build       port:smake
-
-depends_lib         port:gettext
-
-patchfiles          patch-include_schily_sha2.h
-patchfiles-append   mkisofs.c.patch
-post-patch {
-    reinplace -locale C "s|/opt/schily|${prefix}|g" \
-        ${worksrcpath}/DEFAULTS/Defaults.darwin \
-        ${worksrcpath}/DEFAULTS/Defaults.mac-os10 \
-        ${worksrcpath}/DEFAULTS_ENG/Defaults.darwin \
-        ${worksrcpath}/DEFAULTS_ENG/Defaults.mac-os10 \
-        ${worksrcpath}/libfind/find.c \
-        ${worksrcpath}/libfind/find_main.c \
-        ${worksrcpath}/librscg/scsi-remote.c \
-        ${worksrcpath}/TEMPLATES/Defaults.gcc
+PortSystem              1.0
+PortGroup               makefile 1.0
+PortGroup               compiler_blacklist_versions 1.0
+PortGroup               conflicts_build 1.0
+PortGroup               codeberg 1.0
+
+codeberg.setup          schilytools schilytools 2023-04-19
+
+name                    cdrtools
+version                 3.02-${codeberg.version}
+revision                0
+categories              sysutils
+platforms               darwin
+conflicts               dvdrtools
+maintainers             {hotmail.com:amtor @RobK88} \
+                        openmaintainer
+license                 CDDL-1 LGPL-2.1 GPL-2+ GPL-2
+
+description             ISO 9660 filesystem and CD creation tools
+long_description        The cdrtools software includes programs to create \
+                        and/or extract ISO 9660 filesystems, verify their \
+                        integrity, and write them to a disc.
+
+homepage                https://codeberg.org/schilytools/schilytools
+master_sites            ${homepage}/archive/
```

Comment:
pg `codeberg` sets `homepage` and `master_sites`. So go ahead and remove these two lines.

Reply:
Done!